spec/rails_helper.rb

```diff
@@ -156,14 +156,14 @@
   # config.filter_gems_from_backtrace("gem name")
 
   # Explicitly set a seed and time to ensure deterministic Percy snapshots.
-  config.around(:each, percy: true) do |example|
-    Timecop.freeze("2020-05-13T10:00:00Z")
-    prev_random_seed = Faker::Config.random
-    Faker::Config.random = Random.new(42)
+  # config.around(:each, percy: true) do |example|
+  #   Timecop.freeze("2020-05-13T10:00:00Z")
+  #   prev_random_seed = Faker::Config.random
+  #   Faker::Config.random = Random.new(42)
```

Comment:
this can probably remain but let's unblock master first and figure that out later. thanks Molly!